### PR TITLE
feat : alert component implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
alert component done

### CODE REVIEW

File: Components/Alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <link rel="stylesheet" href="../../Docs/doc.css">
+    <link rel="stylesheet" href="alert.css">
+</head>
+<body>
+    <nav class='nav'>
+        <div >
+            <a class="nav-header" href="">Ozone UI</a>
+        </div>
+        <div class="nav-links">
+            <a class='nav-item' href="https://twitter.com/reddygov16">
```
Comment (MaruthiPothuganti) [inline Components/Alert/alert.html]: good to have a target="_blank" for socials

**Review by MaruthiPothuganti — CHANGES_REQUESTED** [review]
Suggested a point. Everything else looks good.

File: Components/Alert/alert.html (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Alert/alert.html]: sure

Comment (govardhanreddyc79) [reply Components/Alert/alert.html]: done

**Pull request merged**